app/serializers/api/participant_serializer.rb

```diff
@@ -77,7 +77,7 @@ class AttributesSerializer < Blueprinter::Base
               targeted_delivery_funding_eligibility: application.targeted_delivery_funding_eligibility,
               withdrawal: withdrawal(application:, lead_provider: options[:lead_provider]),
               deferral: deferral(application:, lead_provider: options[:lead_provider]),
-              created_at: application.accepted_at.rfc3339,
+              created_at: application.accepted_at&.rfc3339,
```

Comment:
sorry this needs to be always there, can you revert, we're backfilling the data, sorry :)

Reply:
🤦